### PR TITLE
hot fix version from Dan at xpd computer

### PR DESCRIPTION
**Pull request by chiahaoliu**
deal with image comes from tiff-squashing

### CODE REVIEW

**Comment by danielballan** [comment]
Instead of maintaining separate fixed and broken varieties, I would prefer to apply the fixes directly to the original implementation.

File: filestore/handlers.py [hunk 1]
```diff
@@ -70,7 +70,52 @@ def __call__(self, point_number):
             raise IntegrityError("Seeking Frame {0} out of {1} frames.".format(
                 stop, len(self._image_sequence)))
         return np.asarray(list(self._image_sequence[start:stop])).squeeze()
+        
+        
+        
+        """
+This is a 'hot fix' of an issue that really needs to be fixed in pims.
```
Comment (danielballan) [inline filestore/handlers.py]: This comment should not be part of the patch.

File: filestore/handlers.py [hunk 2]
```diff
@@ -70,7 +70,52 @@ def __call__(self, point_number):
...
+            # date list of the existing files
+            self._image_sequence = pims.ImageSequence(self._filename)
+            if stop > len(self._image_sequence):
+                # if we _still_ don't have enough files, raise
+                raise IntegrityError("Seeking Frame {0} out of {1} "
+                " frames.".format(stop, len(self._image_sequence)))
+        return np.asarray(list(self._image_sequence[start:stop])).squeeze()
+
+
+import databroker  # so that built-in handlers are registered
```
Comment (danielballan) [inline filestore/handlers.py]: This registration logic should not be part of the patch.

File: filestore/handlers.py [hunk 3]
```diff
@@ -70,7 +70,52 @@ def __call__(self, point_number):
...
+    specs = {'AD_TIFF'} | HandlerBase.specs
+    def __init__(self, fpath, template, filename, frame_per_point=1):
+        self._path = fpath
+        self._fpp = frame_per_point
+        self._template = template.replace('_%6.6d', '*')
+        self._filename = self._template % (self._path,
+                filename)
+        self._image_sequence = pims.ImageSequence(self._filename)
+    def __call__(self, point_number):
+        start = int(point_number * self._fpp)
```
Comment (tacaswell) [inline filestore/handlers.py]: Shoudln't this change just be applied to the existing tifffile handler?  There is no reason that these should ever be floats.

**Comment by tacaswell** [comment]
@chiahaoliu What is the state of this PR?

**Comment by chiahaoliu** [comment]
this is relatively old and outdated one and I think a fix has been implemented already at XPD. I will close it.
